flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BaseWriter.java

```diff
@@ -134,6 +144,15 @@ public void close() {
         if (writeClient != null) {
             writeClient.close();
         }
+        if (numRecordsInSinceChkptCounter != null) {
+            numRecordsInSinceChkptCounter = null;
+        }
+        if (successfullyAppendedRecordsSinceChkptCounter != null) {
+            successfullyAppendedRecordsSinceChkptCounter = null;
+        }
+        if (successfullyAppendedRecordsCounter != null) {
+            successfullyAppendedRecordsCounter = null;
+        }
```

Comment:
Is there no better way of closing a counter?
If not, then do we need to explicitly set it to null?

Reply:
Removed the explicit assignment.